frontend/src/pages/TermPlanner/OptionsHeader/OptionsHeader.tsx

```diff
@@ -28,7 +28,9 @@ const OptionsHeader = () => {
   const planner = plannerQuery.data;
 
   const { theme } = useSelector((state: RootState) => state.settings);
-  const { showMarks, showWarnings } = useSelector((state: RootState) => state.settings);
+  const { showMarks, showPastWarnings: showWarnings } = useSelector(
```

Comment:
it might be good to not rename this, just for consistency

Reply:
Once again, trolled by VSCode automatic rename...